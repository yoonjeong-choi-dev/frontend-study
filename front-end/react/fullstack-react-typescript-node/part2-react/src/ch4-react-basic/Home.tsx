import React, { FC } from "react";

const Home: FC = () => {
  return (
    <div>
      Hello World!
      <p> This is Home Page </p>
    </div>
  );
};

export default Home;