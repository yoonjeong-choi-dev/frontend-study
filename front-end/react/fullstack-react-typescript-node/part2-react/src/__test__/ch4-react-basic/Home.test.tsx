import React from 'react';
import { render } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";
import Home from "../../ch4-react-basic/Home";

describe('Test Home', () => {
  it('render without crashing', () => {
    const { baseElement } = render(<Home/>);
    expect(baseElement).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    const { baseElement } = render(<Home/>);
    expect(baseElement).toMatchSnapshot();
  });
})