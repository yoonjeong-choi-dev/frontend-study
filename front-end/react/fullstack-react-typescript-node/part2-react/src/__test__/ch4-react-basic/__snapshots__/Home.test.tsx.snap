// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home matches snapshot 1`] = `
<body>
  <div>
    <div>
      Hello World!
      <p>
         This is Home Page 
      </p>
    </div>
  </div>
</body>
`;
